lib/auth0/api/v2/actions.rb

```diff
@@ -0,0 +1,201 @@
+module Auth0
+  module Api
+    module V2
+      # Methods to use the actions endpoints
+      module Actions
+        include Auth0::Mixins::Validation
+
+        attr_reader :actions_path
+
+        # Get all actions.
+        # @see https://auth0.com/docs/api/management/v2#!/Actions/get_actions
+        # @param trigger_id [string] An actions extensibility point.
+        # @param action_name [string] The name of the action to retrieve.
+        # @param deployed [boolean] filter to only retrieve actions that are deployed.
+        # @param per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
+        # @param page [integer] The page number. Zero based.
+        # @param installed [boolean] When true, return only installed actions. When false, return only custom actions. Returns all actions by default.
+        # @return [json] Actions and pagination info
+        def actions(trigger_id: nil, action_name: nil, deployed: nil, per_page: nil, page: nil, installed: nil)
+          raise Auth0::MissingTriggerId, 'Must supply a valid trigger_id' if trigger_id.to_s.empty?
+          raise Auth0::MissingActionName, 'Must supply a valid action_name' if action_name.to_s.empty?
+
+          request_params = {
+            trigger_id: trigger_id,
+            action_name: action_name,
+            deployed: deployed,
+            per_page: per_page,
+            page: page,
+            installed: installed
+          }
+          path = "#{actions_path}/actions"
+          get(path, request_params)
+        end
+        alias get_actions actions
+
+        # Create a new action.
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/post_action
+        # @param body [hash] See https://auth0.com/docs/api/management/v2/#!/actions/post_action for available options
+        # @return [json] Returns the created action.
+        def create_action(body = {})
+          post(actions_path, body)
+        end
+
+        # Retrieve the current status of the actions service.
+        # @see https://auth0.com/docs/api/management/v2/#!/actions/get_service_status
+        # @param action_id [string] The action_id of the user to retrieve.
+        #
+        # @return [json] Returns status of the action
+        def actions_status
```

Comment:
Per our discussion with the ecosystems team, we should not add this endpoint to our SDKs.